src/shared/browser-password-bridge-script.test.ts

```
@@ -0,0 +1,95 @@
+import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest'
+import {
+  buildBrowserPasswordBridgeScript,
+  buildBrowserPasswordFillCall
+} from './browser-password-bridge-script'
+import { BROWSER_PASSWORD_MESSAGE_PREFIX } from './browser-credential-types'
+
+const TOKEN = 'tok_aaaaaaaaaaaaaaaa'
+
+function runBridge(): void {
+  // eslint-disable-next-line no-eval
+  ;(0, eval)(buildBrowserPasswordBridgeScript({ token: TOKEN, enabled: true }))
```

Biome (2.5.0)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)